### PR TITLE
Add error log and modify the oss domain name

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

File: docs/manual-CN/3.build-develop-and-administration/1.build/build-source-code.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ bash> git clone https://github.com/vesoft-inc/nebula.git
 ```
 bash> source ~/.bashrc
 ```
-#### 步骤 4: 构建
+#### 步骤 4: 构建, 编译出来的是Debug版本
```
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/1.build/build-source-code.md]: #### 步骤 4A: 构建Debug版本

File: docs/manual-CN/3.build-develop-and-administration/1.build/build-source-code.md [hunk 2]
```diff
@@ -80,14 +80,20 @@ bash> make
 bash> sudo make install
 ```
 
+#### 如果要作为版本发布或用于测试性能，请选择编译成Release版本，重新生成makefile
```
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/1.build/build-source-code.md]: #### 步骤 4B: 构建Release版本

File: docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 3]
```diff
@@ -77,7 +77,7 @@ cd ${package_name} && ./install.sh
 bash> source ~/.bashrc
 ```
 
-#### Step 4: build
+#### Step 4: build, the debug version is compiled
```
Comment (whitewum) [inline docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md]: ditto

**Comment by laura-ding** [comment]
jenkins go

File: build_dep.sh [hunk 4]
```diff
@@ -5,22 +5,16 @@
 # This source code is licensed under Apache 2.0 License,
 # attached with Common Clause Condition 1.0, found in the LICENSES directory.
 #
-# step 1: ./build_dep.sh <C|U|N>
+# step 1: ./build_dep.sh [N] # N: no need packages
```
Comment (whitewum) [inline build_dep.sh]: I think if the user provides no parameter, it's better to be the same as --help. let the use confirm the operations.

how about ./build_dep.sh -y & ,.build_dep.sh -n

Comment (laura-ding) [reply build_dep.sh]: The script header already explains the usage, most customers do not use the parameters

**Pull request merged**